support/packages/local-npm/README.md

```diff
@@ -0,0 +1,213 @@
+# @liferay/local-npm
+
+`@liferay/local-npm` is a private package internal to this monorepo that
+provides a CLI tool to leverage local npm repos (like 
+[verdaccio](https://verdaccio.org/)) to publish local versions of development
+packages to be able to test them before the public release is made.
+
+
+## Installation
+
+Simply run `yarn link` inside the project's folder to make the `local-npm` CLI
+executable available in your system.
+
+> 👀 Note that we don't publish versions of this package to the public npm
+> registry, so the only way to install it is to `yarn link` it.
```

Comment:
Nope, you can link anything you want.

The only problem that can arise if you use a non-published package as a dependency of a project is that every `yarn install` you run will fail (even if you have linked it prior to that install). That, obviously, blocks you unless you remove the offending package from `package.json` to be able to install the other dependencies first, then link what you need.

But in this case, we are not affected by that because we don't put this package as a dependency of anything. We just run `yarn link` to make the tool available in the OS's PATH, so that we can run it from the CLI. Think of it as a local `npm install -g`...